source/standards/configuration-management.html.md.erb

```diff
@@ -1,36 +1,61 @@
 ---
 title: Use configuration management
-last_reviewed_on: 2022-09-25
-review_in: 6 months
+last_reviewed_on: 2025-02-10
+review_in: 24 months
 ---
 
 # <%= current_page.data.title %>
 
-Use [configuration management][] to manage, automate and standardise your infrastructure. When using configuration management you store your [infrastructure as code][] in a version control system such as Git.
+Use [configuration management][] to manage, automate and standardise your infrastructure. When using configuration
+management you store your [infrastructure as code][] in a version control system such as Git.
 
-## Puppet
+## Avoided technologies
 
-The use of [Puppet][] at GDS is diminishing as we move more of our infrastructure to containers and higher level services. It's mainly still in use on [GOV.UK](https://github.com/alphagov/govuk-puppet) but this will decline as more services are moved over to AWS EKS.
+### Puppet
 
-If your environment consists of a simple deployment artefact like an [Amazon Machine Image (AMI)][], Puppet may not be necessary, but the process for building that artefact must still be codified and version controlled.
+The use of [Puppet][] at GDS is diminishing as we move more of our infrastructure to containers and higher level
+services. It is mainly still in use on [GOV.UK](https://github.com/alphagov/govuk-puppet) but this will decline as more
```

Comment:
GOV.UK are now off puppet, so you may delete this bit too if you like. Or leave it and I'll do a separate update

Reply:
https://github.com/alphagov/gds-way/pull/949